files/entrypoint.sh

```diff
@@ -2,6 +2,14 @@
 set -e
 export TELEGRAF_CONFIG_PATH=/etc/telegraf/telegraf_${TELEGRAF_CONFIG_LEVEL}.conf
 export TELEGRAF_HOSTNAME=${TELEGRAF_HOSTNAME:"telegraf"}
+# Check if Redis master server is reachable, fallback to Redis slave
+if nc -z -w 1 redis-master 6379; then
+    echo "Redis master server is reachable, so using it"
+    export REDIS_SERVER=${REDIS_SERVER:-"master"}
+elif nc -z -w 1 redis-slave 6379; then
+    echo "Redis slave server is reachable, so using it"
+    export REDIS_SERVER=${REDIS_SERVER:-"slave"}
```

Comment:
you miss the case where none is defined.
It might more coherent to build the whole line : `servers = ["tcp://redis-local:6379","tcp://redis-${REDIS_SERVER}:6379"]` 

in case of nothing reachable, maybe `servers = []` works and then you can validate the whole logic even on a standalone